README.md

```diff
@@ -15,9 +15,9 @@ Accepts all the arguments and flags `go test` works with.
 Example:
 
 ```
-$ gotest -v github.com/rakyll/hey
+$ gotest -v github.com/jonasbn/go-test-demo
 ```
-![go test output](https://i.imgur.com/udjWuZx.gif)
+![gotest output example screenshot](https://raw.githubusercontent.com/jonasbn/go-test-demo/master/gotest-go-test-demo.png)
```

Comment:
Should probably pin this link to a particular commit to prevent changes on `jonasbn/go-test-demo` master from breaking this repo's example.

```suggestion
![gotest output example screenshot](https://raw.githubusercontent.com/jonasbn/go-test-demo/9daa3c50d60242227a159519873f276cceca4544/gotest-go-test-demo.png)
```

In this case it's not a big issue because I suspect `go-test-demo` won't change much, but it's a bit safer.

Reply:
I have tagged the commit, but I cannot see an easy way to refer to a tag instead.

Reply:
Using the `1.0.0` tag, this url works.

https://raw.githubusercontent.com/jonasbn/go-test-demo/1.0.0/gotest-go-test-demo.png